tests/README.rst

```diff
@@ -1,10 +1,18 @@
 Testing Anaconda
 ================
 
-Testing locally
----------------
+This document describes how to run Anaconda tests. Anaconda has various tests for example
+unit tests, rpm tests, translation tests. All the tests will be run together if you follow
```

Comment:
I suggest:
"rpm tests, translation tests." -> "rpm tests or translation tests."

Reply:
It should be `and` but yes I'll change that.